x-pack/plugins/observability_solution/infra/public/components/logging/log_minimap/log_minimap.tsx

```diff
@@ -51,133 +48,130 @@ function calculateYScale(start: number | null, end: number | null, height: numbe
     .range([0, height]);
 }
 
-export class LogMinimap extends React.Component<LogMinimapProps, LogMinimapState> {
-  constructor(props: LogMinimapProps) {
-    super(props);
-    this.state = {
-      timeCursorY: 0,
-      target: props.target,
-    };
-  }
-
-  public handleClick: React.MouseEventHandler<SVGSVGElement> = (event) => {
+export const LogMinimap = ({
+  start,
+  end,
+  className,
+  height,
+  highlightedInterval,
+  jumpToTarget,
+  summaryBuckets,
+  summaryHighlightBuckets,
+  width,
+  target,
```

Comment:
For a faithful conversion of the original class component we would have to capture the initial `target` in a `useState` so it doesn't update after the first render. I'm not sure about the reason anymore, but I think it has something to do with deep links to specific points in time.

Reply:
I was confused about that and really didn't see any difference with storing `target` in a `useState`, that's why I didn't do it here. `target` isn't changed in this component. 

I'll change it because I don't whether this will have other implications I'm not aware of.